msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -277,6 +277,11 @@ public async Task RemoveAsync(IAccount account)
         internal async Task<AuthenticationResult> AcquireTokenSilentCommonAsync(Authority authority,
             IEnumerable<string> scopes, IAccount account, bool forceRefresh, ApiEvent.ApiIds apiId)
         {
+            if (account == null)
```

Comment:
This is part of a different change and should not be here. I think you forgot to rebase all changes to the current dev. Still, this should not be here.

Reply:
i had to add this for testing, but I did rebase w/current dev, but maybe because i added it earlier (for testing), the changes w/dev were not accounted for?

---
In reply to: [219454270](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1261#discussion_r219454270) [](ancestors = 219454270)